index.html

```diff
@@ -81,235 +81,215 @@
       some condition is met.</li>
       <li>Fetching resources with `XMLHttpRequest` to avoid above behavior
       incurs a serious performance penalty by hiding resource declarations from
-      the user agent's DOM parser and preload scanners. The resource fetches
-      are only dispatched when the relevant JavaScript is executed, which due
-      to abundance of blocking scripts on most pages introduces significant
-      delays and affects application performance.</li>
+      the user agent's DOM and preload parsers. The resource fetches are only
+      dispatched when the relevant JavaScript is executed, which due to
+      abundance of blocking scripts on most pages introduces significant delays
+      and affects application performance.</li>
     </ul>
-    <p>The <a>preload</a> relationship of the HTML Link Element provides a
-    declarative fetch primitive that addresses the above use case of initiating
-    an early fetch and separating fetching from resource execution. As such,
-    <a>preload</a> relationship serves as a low-level primitive that enables
+    <p>The <a>preload</a> keyword on [link] elements provides a declarative
+    fetch primitive that addresses the above use case of initiating an early
+    fetch and separating fetching from resource execution. As such,
+    <a>preload</a> keyword serves as a low-level primitive that enables
```

Comment:
For all intents and purposes they express the same meaning, but I'm aligning language to be (more) similar to that in the HTML spec.. to avoid any further confusion.